README.md

```diff
@@ -1,156 +1,171 @@
+
+For support and discussions visit the [community forum](https://community.dhis2.org/).
+
+For general info visit the [project web page](https://www.dhis2.org/).
+
+For OpenAPI documentation visit the [Stoplight workspace](https://dhis2.stoplight.io/).
+
+For software developer resources visit the [developer portal](https://developers.dhis2.org/).
+
+To contribute to the software read the [contributor guidelines](https://developers.dhis2.org/community/contribute).
+
+The software is open source and released under the [BSD 3-Clause license](https://opensource.org/license/bsd-3-clause).
+
+## Run DHIS2 in Docker
+
+The following guides runs DHIS2 with [Docker Compose](https://docs.docker.com/compose/install/).
+
+A database dump is downloaded automatically the first time you start DHIS2. If you switch between different DHIS2 versions or need to download a different DB dump, you will need to remove the shared volume `db-dump` with the following command.
+
+```sh
+docker compose down --volumes
+```
+
+### Pre-built images
+
+We push pre-built DHIS2 Docker images to Dockerhub. You can pick an `<image name>` from one of the following
+repositories:
+
+* [`dhis2/core`](https://hub.docker.com/r/dhis2/core) - images of the release and release-candidate DHIS2 versions. These images represent the stable DHIS2 versions, meaning they won't be rebuilt in the future.
+
+* [`dhis2/core-dev`](https://hub.docker.com/r/dhis2/core-dev) - images of _the latest development_ DHIS2 versions - branches `master` (tagged as `latest`) and the previous 3 supported major versions. Image tags in this repository will be overwritten multiple times a day.
+
+* [`dhis2/core-canary`](https://hub.docker.com/r/dhis2/core-canary) - images of _the latest daily development_ DHIS2 versions. We tag the last `core-dev` images for the day and add an extra tag with a "yyyyMMdd"-formatted date, like `core-canary:latest-20230124`.
+
+* [`dhis2/core-pr`](https://hub.docker.com/r/dhis2/core-pr) - images of PRs made from [dhis2-core](https://github.com/dhis2/dhis2-core/) and not from forks, as forks do not have access to our organizational secrets.
+
+To run DHIS2 from latest `master` branch (as it is on GitHub) run the command below.
+
+```sh
+DHIS2_IMAGE=dhis2/core-dev:latest docker compose up
+```
+
+### Local image
+
+Build a DHIS2 Docker image as described in [Docker image](#docker-image) and execute the following command.
+
+```sh
+docker compose up
+```
+
+DHIS2 will become available at `http://localhost:8080` with the Sierra Leone Demo DB.
+
+### Demo DB
+
+If you want to start DHIS2 with a specific demo DB you can pass a URL like the below.
+
+```sh
+DHIS2_DB_DUMP_URL=https://databases.dhis2.org/sierra-leone/2.39/dhis2-db-sierra-leone.sql.gz \
+docker compose up
+```
+
+### Launch with Doris
+
+```sh
+DHIS2_IMAGE=dhis2/core-dev:latest DORIS_VERSION=3.0.4 docker compose -f docker-compose.yml -f docker-compose.doris.yml up
+```
+
+#### Generate tables (using default DHIS2 password)
```

Comment:
Not sure this is necessary, as this is not specific to Doris

Reply:
I'll remove it